### ISSUE TITLE
Same plots for loss even with different seeds

### ISSUE THREAD

**Issue by Alexkkir** [chaofengc/IQA-PyTorch#110]
Hello! I ran 4 experiments and they turned out to have very similar charts for the loss function. Even though I used different seeds. What could be the reason?

I run code with command:
```bash
CUDA_VISIBLE_DEVICES=0 python pyiqa/train.py -opt options/train/DBCNN_exps/multitask/vanilla.yml --debug
```
![image](https://github.com/chaofengc/IQA-PyTorch/assets/43682987/7f32b6a6-cc61-4d38-afa2-dc552a81afc3)
<details><summary>Config</summary>
<p>

```yaml
# general settings
# name: debug_DBCNN_LIVEC
name: 006_MultitaskExp_vanilla
model_type: DBCNNModel
num_gpu: 1  # set num_gpu: 0 for cpu mode
# manual_seed: 123

# dataset and data loader settings
datasets:
  train:
    name: koniq10k
    type: GeneralSalDataset
    dataroot_target: ./datasets/koniq10k/512x384
    dataroot_sal: ./datasets/koniq10k/saliency_maps
    meta_info_file: ./datasets/meta_info/meta_info_KonIQ10kDataset.csv
    split_file: ./datasets/meta_info/koniq10k_official.pkl
    mos_range: [0, 100]
    lower_better: false 
    mos_normalize: true

    augment:
      hflip: true
      random_crop: 384
    img_range: 1

    # data loader
    use_shuffle: true
    num_worker_per_gpu: 4
    batch_size_per_gpu: 8
    dataset_enlarge_ratio: 1
    prefetch_mode: ~

  val_clive:
    name: livechallenge
    type: LIVEChallengeDataset
    dataroot_target: ./datasets/LIVEC
    meta_info_file: ./datasets/meta_info/meta_info_LIVEChallengeDataset.csv
    dataroot_sal: ./datasets/LIVEC/saliency_maps
    # split_file: ./datasets/meta_info/livechallenge_seed123.pkl
    # split_index: 2 # which split to use, start from 1

    num_worker_per_gpu: 4
    batch_size_per_gpu: 8
    dataset_enlarge_ratio: 1

  val_koniq:
    name: koniq10k
    type: GeneralSalDataset
    dataroot_target: ./datasets/koniq10k/512x384
    dataroot_sal: ./datasets/koniq10k/saliency_maps
    meta_info_file: ./datasets/meta_info/meta_info_KonIQ10kDataset.csv
    split_file: ./datasets/meta_info/koniq10k_official.pkl

    num_worker_per_gpu: 4
    batch_size_per_gpu: 8
    dataset_enlarge_ratio: 1

# network structures
network:
  type: DBCNNSal
  pretrained: false
  sal_mode: input

# path
path:
  pretrain_network_g: ~
  strict_load_g: true
  resume_state: ~

# training settings
train:
  optim:
    type: SGD
    lr: !!float 1e-3
    momentum: 0.9
    weight_decay: !!float 5e-4

  optim_finetune:
    type: Adam
    lr: !!float 1e-5
    weight_decay: !!float 5e-4

  scheduler:
    type: MultiStepLR
    milestones: [1000]
    gamma: 1

  scheduler_finetune:
    type: MultiStepLR
    milestones: [1000]
    gamma: 1

  total_iter: 20000 # 883 iter/epoch, train/finetune for about 30 epochs
  finetune_start_iter: 10000
  warmup_iter: -1  # no warm up

  # losses
  mos_loss_opt:
    type: MSELoss
    loss_weight: !!float 1.0

  # sal_loss_opt:
  #   type: MSESalLoss
  #   loss_weight: !!float 0.001

  # multitask_loss_opt:
  #   type: MultiTaskLoss
  #   n_tasks: 2
  
# validation settings
val:
  val_freq: !!float 800
  save_img: false
  pbar: true

  key_metric: srcc # if this metric improve, update all metrics. If not specified, each best metric results will be updated separately
  metrics:
    srcc:
      type: calculate_srcc

    plcc:
      type: calculate_plcc

    krcc:
      type: calculate_krcc

# logging settings
logger:
  print_freq: 100
  save_latest_freq: !!float 5e2
  use_tb_logger: true
  wandb:
    project: IQA-7 pyiqa
    resume_id: ~

# dist training settings
dist_params:
  backend: nccl
  port: 29500

``` 

</p>
</details> 

* [wandb report](https://wandb.ai/alexkkir/IQA-7%20pyiqa/reports/Similar-plots-for-different-runs--Vmlldzo1NjEwMDc1)
* [my version of code](https://github.com/Alexkkir/pyiqa-sal)
